lib/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/impl/HistoryQueryImpl.java

```diff
@@ -0,0 +1,641 @@
+package pro.taskana.simplehistory.impl;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import pro.taskana.ClassificationQuery;
+import pro.taskana.TimeInterval;
+import pro.taskana.exceptions.InvalidArgumentException;
+import pro.taskana.BaseQuery.SortDirection;
+import pro.taskana.history.api.TaskHistoryEvent;
```

Comment:
you don't use SortDirection, TaskHistoryEvent and ClassificationQuery, can you remove it?